packages/cf.js/test/app-instance.spec.ts

```diff
@@ -100,4 +101,116 @@ describe("CF.js AppInstance", () => {
       await appInstance.uninstall();
     });
   });
+
+  describe("Node events", () => {
+    it("fires update state events", async () => {
+      expect.assertions(4);
+
+      const testOldState = bigNumberify(1000);
```

Comment:
Do you find the prefix `test` helpful in these variable names? I'd think it'd be implied given that this is in the context of a test

Just implicit/explicit comparison thoughts

What do you think about the `expected` prefix?

This would lead to:
```
expect(action).toBe(expectedAction)
```

Reply:
Yeah that reads nicer, just a bit long.